### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by etrombly**
add support for windows builds

### CODE REVIEW

**Comment by TheDarkula** [comment]
I really appreciate the energy you put into this. The only thing is, I created this specifically for linux (hence the use of gtk). So I'll only be supporting linux builds.

**Comment by gregkrsak** [comment]
Well....does it run properly in Windows with these changes or not?

**Comment by TheDarkula** [comment]
It still doesn't run properly even with these changes, because the numpad doesn't work for entry and the css styling is very off.

**Comment by gregkrsak** [comment]
Ok. Hey, it's your project. You'd get a lot more widespread adoption if you made this cross-platform.


But a guy (you) smart enough to make something like this already knows that. So, I'm wondering why you *really* only want a limited subset of Linux users [see issue #1] to use your New Coolest Thing.


Not trying to be rude. It just seems like you're aware of your own limitations. Which, in reality and as an ideal, is a good thing. But. I think, moving forward, you should clearly communicate the limitations of this project when you introduce it to new folks. Most people aren't Linux people. 


You should embrace the work of other contributors, even if you have to make a separate unstable windows branch.

**Comment by marleybrush** [comment]
Other owner here.

You are correct, this is our project and we created this program because we are brewers who needed a tool that's accurate and in one location. Also, one that is not dependent on a website that could potentially go down at any moment. 

We shared this project so others could check out Rust and GTK, provide ideas, and even use it themselves if they wanted to do so. We have not only embraced, but appreciated other people's input. There were two suggestions/corrections about the code, which I greatly appreciate because it makes the program better.

We've worked really diligently at using the best formulas, writing the most efficient code, and creating a really cool looking theme from scratch.

Bottom line, I'm so glad we shared this project because it's better now with other people's input.

**Comment by gregkrsak** [comment]
Hey, I'm not insulting your program. Just providing professional feedback. 


You publicly released released a very cool little app that didn't work (at all) for the vast majority of the users who you presented it to on Reddit. 


I hope your software succeeds, your team grows, and you all continue to improve your abilities and communication.
